survey_form/index.html

```diff
@@ -43,8 +43,94 @@
 
     <div class="tittle"><h1 class="tittle__item">Survey Form</h1></div>
 
-    <form class="form">
-      <p class="form__tittle">Lets us now how we can improve freeCodeCamp</p>
+    <form class="form-survey">
+      <legend class="form-survey__title">
+        Lets us now how we can improve freeCodeCamp
+      </legend>
+
+      <label class="form-survey__label" for="name">Name: </label>
+      <input
+        type="text"
+        id="name"
+        class="form-survey__input"
+        required
+        placeholder="Leonardo DiCaprio"
+      />
+
+      <label class="form-survey__label" for="email">Email: </label>
+      <input
+        type="email"
+        id="email"
+        class="form-survey__input"
+        required
+        placeholder="leonardo.dicaprio@gmail.com"
+      />
+
+      <label class="form-survey__label" for="age">Age: </label>
+      <input
+        type="age"
+        id="number"
+        class="form-survey__input"
+        placeholder="44"
+        min="12"
+        max="123"
+        required
+      />
+
+      <label class="form-survey__label" for="current_role"
+        >Which option best describes your current role?</label
+      >
+
+      <select class="form-survey__dropdown" id="current_role">
+        <option value="select_an_option" disabled>Select an option</option>
+        <option value="student">Student</option>
+        <option value="full_time_job">Full time job</option>
+        <option value="full_time_learner">Full time learner</option>
+        <option value="prefer_not_to_say">Prefer not to say</option>
+        <option value="other">Other</option>
+      </select>
+
+      <label
+        class="form-survey__label form-survey__label--radiobutton"
+        for="form-survey__radiobutton"
```

Comment:
Wcześniej to jeszcze nie wyszło, ale ten label nie odnosi sie do żadnego inputa/selecta bezpośrednio, więc nie powinien mieć atrybutu `for`. Pełni funkcję bardziej opisową. Ewentualnie można zastąpić tagiem `legend`.